src/mlpack/core/data/save.hpp

```diff
@@ -90,6 +91,22 @@ bool Save(const std::string& filename,
           const bool fatal = false,
           format f = format::autodetect);
 
+#ifdef HAS_STB
+template<typename eT>
+bool Save(const std::string& filename,
+          arma::Mat<eT>& matrix,
+          ImageInfo& info,
+          const bool fatal = false,
+          const bool transpose = true);
+
+template<typename eT>
+bool Save(const std::vector<std::string>& files,
+          arma::Mat<eT>& matrix,
+          ImageInfo& info,
+          const bool fatal = false,
+          const bool transpose = true);
+#endif // HAS_STB.
```

Comment:
Same comment here as in `Load()` when `HAS_STB` is not defined.

Reply:
Okay.

Reply:
@MuLx10 Thanks---let us know when you're done with further changes for this PR.  I think then we can merge it. :+1: